packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -9,15 +9,15 @@ const TradersHubOnboarding = observer(() => {
     const history = useHistory();
     const { traders_hub, ui } = useStore();
     const { setIsOnboardingVisited } = traders_hub;
-    const { is_dark_mode_on } = ui;
+    const { is_dark_mode_on, is_mobile } = ui;
 
     return (
         <div data-testid='dt_traders_hub_onboarding'>
             <div className='traders-hub-header__tradershub--onboarding--logo'>
                 <Popover
                     classNameBubble='account-settings-toggle__tooltip'
                     alignment='bottom'
-                    message={<Localize i18n_default_text='View onboarding' />}
+                    message={!is_mobile && <Localize i18n_default_text='View onboarding' />}
```

Comment:
In mobile, we don't need a popover tooltip for this icon that redirects to Onboarding since there is no hover on mobile, and it should redirect immediately as per requirements.
Note: Even with this popover, it was always working for Android but not for iOS. Hence, removing to have consistent behavior across devices.